src/path_info.h

```diff
@@ -5,17 +5,109 @@
 #include <map>
 #include <string>
 
-extern std::map<std::string, std::string> FILENAMES;
+class Path
```

Comment:
Seems more like this represents a set of paths, not a single path. A path is something like a string "a/b/c".

Reply:
You're right, it was actually the first name I had in mind at the moment, but I'm open to suggestions.

Maybe: PathsApp or PathsCata or Filenames, etc ....

------

_This has been made possible by Deepl._